proposed/event-dispatcher-meta.md

```diff
@@ -37,85 +37,84 @@ The Working Group identified four possible workflows for event passing, based on
 
-### 4.2 Intended use cases
+### 4.2 Example applications
 
-Despite their similarities, the Message and Task pipelines have two distinct workflows and use cases.
-
-| Messages                      | Tasks
-|-------------------------------|-------------------------------
-| Must be immutable             | May be mutable
-| Must be serializable          | May be serializable
-| Listeners may be delayed      | Listeners must be called synchronously immediately
-| Listener order not guaranteed | Listeners must be called in the order returned from the Provider
-| All listeners will fire       | Listeners may short-circuit the pipeline (if the Stoppable interface is implemented)
-| One-way communication only    | May be one-way or two-way communication
-
-As a general guideline, a Message Notifier is appropriate when:
-
-* The Emitter does not care what actions listeners take in response to an event.
-* The common case would be that only a single listener is registered, that is, when the Message Notifier is being used as a Command Bus.
-* The Emitter anticipates listeners to be expensive and therefore likely to be deferred for later processing.
-
-A Task Processor is appropriate when:
-
-* The event is being used as a "hook" or "pointcut" to extend or modify the Emitter's behavior.
-* The Emitter wishes to allow listeners to interact with each other (through mutating the event).
-* The Emitter wishes to allow a listener to terminate the pipeline early before other listeners have completed.
-
-If uncertain which is appropriate, the Task Processor offers more functionality and is therefore the safer default.
+* Indicating that some change in system configuration or some user action has occurred and allowing other systems to react in ways that do not affect program flow (such as sending an email or logging the action).
+* Passing an object to a series of Listeners to allow it to be modified before it is saved to a persistence system.
+* Passing a collection to a series of Listeners to allow them to register values with it so that the Emitter may act on all of the collected information.
+* Passing a collection to a series of Listeners to allow them to modify the collection in some way before the Emitter takes action.
+* Passing some contextual information to a series of Listeners so that all of them may "vote" on what action to take, with the Emitter deciding based on the aggregate information provided.
+* Passing an object to a series of listeners and allowing one of them to set a value and then prevent further listeners from running.
```

Comment:
The phrasing in the original was better here. Might I suggest: "Passing an object to a series of listeners, and allowing any listener to terminate early before other listeners have completed."

Reply:
At this point we put in words like "Emitter" which hasn't been introduced yet. I think going with "Dispatcher" might be easier.

Reply:
"Emitter" is defined in the spec itself to mean "The thing that creates an event and passes it to the dispatcher".  Dispatcher being "The thing that implements DispatcherInterface".

Updated per MWOP, although I included "the process" as the thing being terminated.